sql-plugin/src/main/scala/org/apache/spark/sql/rapids/GpuCartesianProductExec.scala

```diff
@@ -190,10 +218,10 @@ object GpuNoColumnCrossJoin extends Arm {
     // Hash aggregate explodes the rows out, so if we go too large
     // it can blow up. The size of a Long is 8 bytes so we just go with
     // that as our estimate, no nulls.
-    val maxRowCount = targetSizeBytes/8
+    val maxRowCount = targetSizeBytes / 8
```

Comment:
In the future, please don't make a lot of these whitespace changes throughout the file.

Reply:
Sorry, I will be careful not to do this in next time.